server.go

```diff
@@ -242,6 +242,11 @@ func KeepaliveEnforcementPolicy(kep keepalive.EnforcementPolicy) ServerOption {
 // CustomCodec returns a ServerOption that sets a codec for message marshaling and unmarshaling.
 //
 // This will override any lookups by content-subtype for Codecs registered with RegisterCodec.
+//
+// Deprecated: register codecs using encoding.RegisterCodec. The server will
+// automatically use registered codecs based on the incoming requests' headers.
+// See also
+// https://github.com/grpc/grpc-go/blob/master/Documentation/encoding.md#using-a-codec.
```

Comment:
Can you look up & see whether this is something we must keep for 1.x or if it's something we may delete in the future?  And add that to the comment here.  (See other "deprecated" comments for examples.)

Reply:
Done. All deprecated APIs in this file were all available at GA.

Reply:
Thanks.